domain/src/test/java/org/oppia/android/domain/topic/StoryProgressTestHelperTest.kt

```diff
@@ -790,87 +795,116 @@ class StoryProgressTestHelperTest {
     )
     testCoroutineDispatchers.runCurrent()
 
-    topicListController.getOngoingStoryList(profileId).toLiveData()
-      .observeForever(mockOngoingStoryListObserver)
+    topicListController.getPromotedActivityList(profileId).toLiveData()
+      .observeForever(mockPromotedActivityListObserver)
     testCoroutineDispatchers.runCurrent()
 
-    verifyGetOngoingStoryListSucceeded()
+    verifyGetPromotedActivityListSucceeded()
 
-    val ongoingStoryList = ongoingStoryListResultCaptor.value.getOrThrow()
-    assertThat(ongoingStoryList.recentStoryCount).isEqualTo(2)
-    assertThat(ongoingStoryList.olderStoryCount).isEqualTo(0)
-    assertThat(ongoingStoryList.recentStoryList[0].explorationId).isEqualTo(
-      RATIOS_EXPLORATION_ID_0
-    )
-    assertThat(ongoingStoryList.recentStoryList[0].completedChapterCount).isEqualTo(0)
+    val promotedActivityList = promotedActivityListResultCaptor.value.getOrThrow()
+    with(promotedActivityList.promotedStoryList) {
+      assertThat(recentlyPlayedStoryCount).isEqualTo(2)
+      assertThat(olderPlayedStoryCount).isEqualTo(0)
+      assertThat(recentlyPlayedStoryList[0].explorationId).isEqualTo(
+        RATIOS_EXPLORATION_ID_0
+      )
+      assertThat(recentlyPlayedStoryList[0].completedChapterCount).isEqualTo(0)
+      assertThat(recentlyPlayedStoryList[1].explorationId).isEqualTo(
+        RATIOS_EXPLORATION_ID_2
+      )
+      assertThat(recentlyPlayedStoryList[1].completedChapterCount).isEqualTo(0)
+    }
+  }
 
-    assertThat(ongoingStoryList.recentStoryList[1].explorationId).isEqualTo(
-      RATIOS_EXPLORATION_ID_2
+  @Test
+  fun testProgressTestHelper_markFullProgressForSecondTestTopic_promotedStoryListIsCorrect() {
+    storyProgressTestHelper.markFullProgressForSecondTestTopic(
+      profileId = profileId,
+      timestampOlderThanAWeek = false
     )
-    assertThat(ongoingStoryList.recentStoryList[1].completedChapterCount).isEqualTo(0)
+    testCoroutineDispatchers.runCurrent()
+
+    topicListController.getPromotedActivityList(profileId).toLiveData()
+      .observeForever(mockPromotedActivityListObserver)
+    testCoroutineDispatchers.runCurrent()
+
+    verifyGetPromotedActivityListSucceeded()
+
+    val promotedActivityList = promotedActivityListResultCaptor.value.getOrThrow()
+    with(promotedActivityList.promotedStoryList) {
+      assertThat(recentlyPlayedStoryCount)
+        .isEqualTo(0)
+      assertThat(olderPlayedStoryCount)
+        .isEqualTo(0)
+      assertThat(suggestedStoryList[0].explorationId)
+        .isEqualTo(
+          FRACTIONS_EXPLORATION_ID_0
+        )
+    }
   }
 
   @Test
-  fun testProgressTestHelper_markRecentlyPlayed_firstStoryInTestTopic1And2_storyListIsCorrect() {
+  fun testProgressTestHelper_markRecentlyPlayed_firstStoryInTestTopic1And2_promotedStoryListIsCorrect() { // ktlint-disable max-line-length
```

Comment:
testProgressTestHelper_markRecentlyPlayed_testTopic1And2_promotedStoryListIsCorrect

Reply:
Renamed to testProgressTestHelper_markRecentlyPlayed_firstStoryInTestTopic1And2_promotedListIsCorrect()